src/test/java/org/opentripplanner/OtpArchitectureModules.java

```diff
@@ -10,6 +10,8 @@ public interface OtpArchitectureModules {
   Package JACKSON_ANNOTATIONS = Package.of("com.fasterxml.jackson.annotation");
   Package JTS_GEOM = Package.of("org.locationtech.jts.(*)..");
 
+  Package GUAVA = Package.of("com.google.common.(*)..");
```

Comment:
Should we allow only explicit things from guava, such as multimap, (and not deprecated things like `Lists.newArrayList()`)

Reply:
created issue #4325